cmd/scw/testdata/test-all-usage-rdb-certificate-get-usage.golden

```diff
@@ -1,19 +0,0 @@
-🎲🎲🎲 EXIT CODE: 0 🎲🎲🎲
```

Comment:
Strange that these rdb goldens are deleted 🤔

Reply:
command is deleted